osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -445,6 +445,15 @@ protected override Task Poll()
             return tcs.Task;
         }
 
+        public void MarkChannelAsRead(Message message)
```

Comment:
This is definitely requiring a xmldoc.

Reply:
Resolved with https://github.com/ppy/osu/pull/7425/commits/50e357a79930b0980fb67fddafc21c65712a0c49